activities/twenty_three_and_me/models.py

```diff
@@ -8,6 +6,14 @@
 from data_import.models import BaseDataFile, DataRetrievalTask
 
 
+def get_upload_path(instance, filename=''):
+    """
+    Construct the upload path for a 23andMe upload.
+    """
+    return 'member/{}/uploaded-data/23andme/{}'.format(instance.user.id,
```

Comment:
Idea - maybe timestamp so we don't overwrite files with the same name, from the same user? thoughts?